spec/models/friendship_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Friendship, type: :model do
```

Comment:
[OPTIONAL]

Add tests to check for unique friendship constraint. Two users are only allowed to create 1 unique friendship.